devnotes/pnor_layout.md

```diff
@@ -0,0 +1,186 @@
+| FIRDATA                     | 0x12 | 0x03941000:0x03943FFF  | 0x3000       | d    | -     | same as size | None                                  |
+| [VERSION](#VERSION)         | 0x13 | 0x03944000:0x03945FFF  | 0x2000       | d    | -     | same as size | skiboot verifies this, but just warns about mismatch |
+| BMC_INV                     | 0x14 | 0x03968000:0x03970FFF  | 0x9000       | d    | -     | same as size | None                                  |
+| [HBBL](#HBBL)               | 0x15 | 0x03971000:0x03977FFF  | 0x7000       | d    | -     | same as size | None                                  |
+| ATTR_TMP                    | 0x16 | 0x03978000:0x0397FFFF  | 0x8000       | d    | -     | same as size | None                                  |
+| ATTR_PERM                   | 0x17 | 0x03980000:0x03987FFF  | 0x8000       | d    | -     | same as size | None                                  |
+| [IMA_CATALOG](#IMA_CATALOG) | 0x18 | 0x03989000:0x039C8FFF  | 0x40000      | d    | -     | same as size | None                                  |
+| RINGOVD                     | 0x19 | 0x039C9000:0x039E8FFF  | 0x20000      | d    | -     | same as size | None                                  |
+| WOFDATA                     | 0x1A | 0x039E9000:0x03CE8FFF  | 0x300000     | d    | -     | same as size | None                                  |
+| HB_VOLATILE                 | 0x1B | 0x03CE9000:0x03CEDFFF  | 0x5000       | d    | -     | same as size | None                                  |
+| [MEMD](#MEMD)               | 0x1C | 0x03cee000:0x03CFBFFF  | 0xE000       | d    | -     | same as size | VPD for RAM IIRC                      |
+| SBKT                        | 0x1D | 0x03d02000:0x03D05FFF  | 0x4000       | d    | -     | same as size | None                                  |
+| [HDAT](#HDAT)               | 0x1E | 0x03d06000:0x03D0DFFF  | 0x8000       | d    | -     | same as size | None                                  |
+| UVISOR                      | 0x1F | 0x03d10000:0x03E0FFFF  | 0x100000     | d    | -     | same as size | None                                  |
+| [BOOTKERNFW](#BOOTKERNFW)   | 0x20 | 0x03e10000:0x03FEFFFF  | 0x1E0000     | d    | -     | same as size | None                                  |
+| [BACKUP_PART](#BACKUP_PART) | 0x21 | 0x03ff7000:0x03FFEFFF  | 0x8000       | d    | -     | 0            | Required                              |
+
+> Note: If partition has no coreboot usage means that erasing this partition
+> didn't prevent the platform from starting skiroot.
+> Partition may still be used in some case
+> or in the later stages of the booting process.
+
+Type:
+* l = FFS_TYPE_LOGICAL
+* d = FFS_TYPE_DATA
+* p = FFS_TYPE_PARTITION
+
+Flags:
+* b = FFS_FLAGS_U_BOOT_ENV
+* p = FFS_FLAGS_PROTECTED
+
+# Reproducibility
+Reproducibility of partitions can be found at the
```

Comment:
Rather, content (binary placed in the partition) can be reproducible, not the partition itself.

Reply:
Fixed